com.sap.cloud.lm.sl.cf.web/src/main/java/com/sap/cloud/lm/sl/cf/web/configuration/ActivitiConfiguration.java

```diff
@@ -0,0 +1,96 @@
+package com.sap.cloud.lm.sl.cf.web.configuration;
```

Comment:
I am with a mixed feelings here... :(
On one side, the creation using the Java Spring configuration fills me with happiness :D
but on the other, it is now a little more complicated to create the activiti shits with such Java configuration?

What do you say?

Can we use the Java Configuration in order to make some magic tricks here? As we can do in the FileSystemFileServiceFacotryBean(or w/e was)

But on the other hand, the next commit uses it in order to create the job-id in more precise way. Anyway, I am staying with mixed feelings JUST in this case, otherwise I prefer JAVA CONFIG

Reply:
I'm also not the biggest fan of this class, but it does "fix" a couple of issues, which were present in the XML configuration:
  1. Renaming classes will automatically rename them here as well.
  2. Removes the duplication in the two activiti-context.xml files (one in this project and one in the internal).

The downside is that the class is more verbous than the XML, but considering the other benefits, I'm willing to live with it. :)

Reply:
Yes, I haven't thought about the benefits. I want to live with it, too